types/google.visualization/index.d.ts

```diff
@@ -654,6 +672,11 @@ declare namespace google {
             reverseCategories?: boolean;
             selectionMode?: string // single / multiple
             series?: any;
+            domainAxis?: { type: string };
+            trendlines?: any;
```

Comment:
`trendlines` is type of `object` not `any`.

Reply:
Changed to number keyed list of anys.